examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,43 @@
+import { loadStdlib } from '@reach-sh/stdlib';
+const stdlib = loadStdlib(process.env);
+
+const startingBalance = stdlib.parseCurrency(1000);
+
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 5000000;
+minter.setGasLimit(gasLimit);
+receiver.setGasLimit(gasLimit);
+
+const mintAddr = minter.getAddress();
+console.log(`Your address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who) => fmt(await stdlib.balanceOf(who));
+const bal = await getBal(minter);
+console.log(`Minter starting balance: ${bal}.`);
+
+console.log(`Creating the NFT`);
+const theNFT = await stdlib.launchToken(minter, "nftName", "SYM", { supply: 1, url: "https://reach.sh", clawback: null, freeze: null, defaultFrozen: false, reserve: null, note: Uint8Array[1]});
+const nftId = await theNFT.id;
+
+const preAmtNFT = await minter.balanceOf(nftId);
+console.log(`Minter has ${preAmtNFT} of the NFT`);
+    
+if (stdlib.connector == 'ALGO' && await receiver.tokenAccept(nftId)) {console.log(`Receiver opted-in to NFT`)};
+if (stdlib.connector == 'ALGO' && await receiver.tokenAccepted(nftId)) {console.log(`Token accepted`)};
+await stdlib.transfer(minter, receiver, 1, nftId);
+console.log(`NFT transfer made from minter to receiver`);
+
+const postBal = await getBal(minter);
+console.log(`Minter balance after transfer: ${postBal}.`);
+
+const postAmtNFT = await receiver.balanceOf(nftId);
+console.log(`Receiver has ${postAmtNFT} of the NFT`);
```

Comment:
I think it would be better to structure this like...
- A library (i.e. a single function) that can launch an NFT
- A test case (i.e. a use of the function with the testing code... basically what you have)
- An interactive program that does it live
- A demo "real" program that launches N NFTs with different traits/etc

Reply:
re: the library:
Could I take the `mintNFT` function and wrap it in a `launchNFT` method and add that to the Reach stdlib? 
That'd be neat!
I think the `logBalance` and `transferNFT` functions are too specific to be useful as library methods.